Update README wording about scikit-image example (#8125)
Napari/napari: README.md

## simple example

> This example uses a data sample from the `scikit-image` library, but you can change the example to use your own dataset.

Reviewer: ```suggestion
This example uses a data sample from the `scikit-image` library, but you can pass your own dataset as an array to `imshow`.
```

More explicit?

Reviewer 2: I dig it!
